51.Cafe-bot Prerequisites Steps

### Github issues for [C#](https://github.com/Microsoft/botbuilder-dotnet/issues) /[JS](https://github.com/Microsoft/botbuilder-js/issues) / [Java](https://github.com/Microsoft/botbuilder-java/issues)/ [Python](https://github.com/Microsoft/botbuilder-python/issues) should be used for bugs and feature requests. Use [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework) for general "how-to" questions. 

## Sample information
1. Sample type: samples
2. Sample language: dotnetcore
3. Sample name: 51.Cafe-bot

## Describe the bug
Following prerequisites "Install CLI tools" by running "npm i -g msbot chatdown ludown luis-cli qnamaker" errors out.

## To Reproduce
Steps to reproduce the behavior:
1. Go to 'command line and run the following command: npm i -g msbot chatdown ludown luis-cli qnamaker'

4. See error
> luis-cli@1.0.1 typings C:\Users\maxim\AppData\Roaming\npm\node_modules\luis-cli
> typings install

'typings' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! luis-cli@1.0.1 typings: `typings install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the luis-cli@1.0.1 typings script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\maxim\AppData\Roaming\npm-cache\_logs\2018-12-05T08_01_39_446Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! luis-cli@1.0.1 postinstall: `npm run typings && npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the luis-cli@1.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\maxim\AppData\Roaming\npm-cache\_logs\2018-12-05T08_01_39_922Z-debug.log

## Expected behavior
Running the command should not produce errors.

## Screenshots
If applicable, add screenshots to help explain your problem.

## Additional context
Add any other context about the problem here.

[bug]

--- a/samples/csharp_dotnetcore/51.cafe-bot/README.md
+++ b/samples/csharp_dotnetcore/51.cafe-bot/README.md
@@ -73,7 +73,7 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
 - In a terminal
 - Install required tools
     ```bash
-    npm i -g msbot chatdown ludown luis-cli qnamaker 
+    npm i -g msbot chatdown ludown luis-apis qnamaker 
     ```
 
 ## Configure required services

